daffodil-core/src/main/scala/org/apache/daffodil/grammar/primitives/PrimitivesZoned.scala

```diff
@@ -132,6 +132,7 @@ case class ConvertZonedNumberPrim(e: ElementBase)
       roundingMode,
       roundingIncrement,
       Nil,
+      true,
```

Comment:
Something that IDEA has been suggesting to me is to pass args using the scala convention where you can supply an argument name.  When the args have no defaults like this, and there's a flock of them, it just adds "checked documentation" to the code, because you still have to supply them all, and in the right order. But it improves readability to write 

`isInt = true,`

rather than just true floating out there.

Reply:
Fixed.